storage/watchable_store_test.go

```diff
@@ -222,3 +240,130 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
+
+func TestNewMapWatchingToEventMap(t *testing.T) {
+	var (
+		testN         = 5
```

Comment:
let's remove the test prefix since they are in test already. Also let's remove the var block. just use `:` to assign value while doing declaration.